controls/SV-254523.rb

```diff
@@ -40,7 +40,7 @@
   else
```

Comment:
Note: The `if` condition might lead to a false positive since the default value for `emergency_accounts` is empty.